src/components/CreateGoals.jsx

```diff
@@ -92,23 +71,45 @@ function CreateGoals({installNeeded, user, onRepoCreation}) {
     <React.Fragment>
       <ContextStyle>
         <SpaceBetweenTop>
-          <FlexColumn>{installReady ? <InstallApp user={user} /> : <CreateApp />}</FlexColumn>
-          <Illustration alt="productive developer image" src={devProductive} />
+          <SpaceBetween>
+              <CreateApp />
+          </SpaceBetween>
         </SpaceBetweenTop>
+        <Cards disabled={installReady}>
+          <SpaceBetween>
+            <OnBoardingText>
+              <h1>1</h1>
+              <p>Let's sync Open Sauced with your GitHub Repos</p>
+            </OnBoardingText>
+            <Button primary minWidth={175} maxWidth={175} onClick={_handleRepoCreation} disabled={installReady}>
+              Sync Repos
+            </Button>
+          </SpaceBetween>
+        </Cards>
+        <Cards disabled={!installReady}>
+          <SpaceBetween>
+            <OnBoardingText>
+              <h1>2</h1>
+              <p>Now let's create the Open Sauced database on GitHub</p>
+            </OnBoardingText>
+            <a
+              rel="noreferrer"
+              target="_blank"
+              href={`https://github.com/apps/open-sauced/installations/new/permissions?target_id=${user && user.id}`}>
+              <Button primary minWidth={175} maxWidth={175} disabled={!installReady}>Create database</Button>
+            </a>
+          </SpaceBetween>
+        </Cards>
+        {/* <Cards disabled={true}>
```

Comment:
LGTM except comments :D

Reply:
Ah yeah, I commented out the third card. There's a bit of work that needs to go into making the third card work properly and so I commented it out for this PR instead of deleting it so I can jump back to working on it for the next PR I'm going to work on. Should I delete it completely? Figured it would be okay since I'm going to get to work on it right after finishing this.

Reply:
Okay, no problem. I'll go put that TODO comment into the code. Thanks much.